cdap-docs/reference-manual/source/glossary.rst

```diff
@@ -98,5 +98,16 @@ Glossary
    Apache Hadoop
       See :term:`Hadoop`.
 
+   Master Services
+      CDAP system services that are run in YARN containers like Transaction Service,
+      Dataset Executor, Log Saver, Metrics Processor, etc.
+
+
+.. |(TM)| unicode:: U+2122 .. trademark sign
```

Comment:
Don't add these two replaces, as I have already merged them in as part of the common conf.py used to build the docs.